packages/core/content-manager/server/services/utils/populate.js

```diff
@@ -233,28 +232,17 @@ const getQueryPopulate = async (uid, query) => {
      * @param {string} param0.path - Content Type path to the attribute
      * @returns
      */
-    ({ key, attribute, path }) => {
+    ({ attribute, path }) => {
       // TODO: handle dynamic zones and morph relations
       if (!attribute || isDynamicZone(attribute) || isMorphToRelation(attribute)) {
         return;
       }
 
       // Populate all relations, components and media
       if (isRelation(attribute) || isMedia(attribute) || isComponent(attribute)) {
-        const populatePath = pathToPopulate(path.attribute);
-        populateQuery = set(populatePath, { fields: [] }, populateQuery);
-        return;
+        const populatePath = path.attribute.replace(/\./g, '.populate.');
```

Comment:
What if we have a field named populate? COuld it be possible? :thinking:

Reply:
that is certainly possible, but would it be an issue here?
I will add a test just in case, good point.

Reply:
It is working correctly, added a unit test just in case [`566a0f7` (#16814)](https://github.com/strapi/strapi/pull/16814/commits/566a0f74ba597d75e0f5cbcd59d94b4094669be1)